frontend/src/components/AppointmentSchedule.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import {Calendar, momentLocalizer} from 'react-big-calendar'
+import moment from 'moment'
+import 'react-big-calendar/lib/css/react-big-calendar.css'
+
+const localizer = momentLocalizer(moment)
+
+export default () => {
+
+  return (
+      <div style={{ height: 700, marginTop: 40 }}>
```

Comment:
I prefer to use CSS modules rather than inline styles, but that's not a big problem; we can address it later.